tests/ImageSharp.Benchmarks/Codecs/Jpeg/DecodeJpegParseStreamOnly.cs

```diff
@@ -15,7 +14,7 @@ namespace SixLabors.ImageSharp.Benchmarks.Codecs.Jpeg
     [Config(typeof(Config.ShortClr))]
     public class DecodeJpegParseStreamOnly
     {
-        [Params(TestImages.Jpeg.Baseline.Jpeg420Exif)]
+        [Params(TestImages.Jpeg.BenchmarkSuite.Lake_Small444YCbCr)]
```

Comment:
Updated to match the full decoding benchmark so we can more easily see the difference.